src/bundles/config.js

```diff
@@ -29,12 +32,23 @@ bundle.selectConfigObject = createSelector(
 
 bundle.selectApiUrl = createSelector(
   'selectConfigObject',
-  (config) => getURLFromAddress('API', config) || 'https://ipfs.io'
+  (config) => getURLFromAddress('API', config) || DEFAULT_URL
 )
 
 bundle.selectGatewayUrl = createSelector(
   'selectConfigObject',
-  (config) => getURLFromAddress('Gateway', config) || 'https://ipfs.io'
+  async (config) => {
```

Comment:
I believe this is why E2E fails. If I comment out async/await all tests are green.

I don't feel strong with advice here, but IIUC if you want to have async selector, you need to use [async-selector](https://www.npmjs.com/package/async-selector)/[use-async-selector](https://www.npmjs.com/package/use-async-selector), OR (remove async/await and refactor `checkIfGatewayUrlIsAcessible` to use old school blocking [synchronous XHR request](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Synchronous_and_Asynchronous_Requests#Synchronous_request) (less dependencies, but will produce warning when done in the main thread)

Reply:
Thank you for finding the issue 🙏